core/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/debug/BreakpointManagerImpl.java

```diff
@@ -141,33 +136,15 @@ public void changeBreakpointState(final int lineNumber) {
      * Removes breakpoint mark.
      */
     private void deleteBreakpoint(final VirtualFile activeFile,
-                                  final Debugger debugger,
+                                  @Nullable final Debugger debugger,
```

Comment:
The method's body is clear and there is no need to mark this argument as nullable